package/setup.py

```diff
@@ -67,6 +67,14 @@
     cython_found = False
     cmdclass = {}
 
+# Find our own release code (by shamelessly running MDAnalysis/version.py,
+#  which brings the __version__ variable into the current namespace.)
+with open("MDAnalysis/version.py") as f:
```

Comment:
After I submitted I realized that this is relatively fragile (the tests' `setup.py` can't do the same, for instance). It was mostly a convenience so that you'd only need to change the version string in one place. It's not even directly related to the issue at hand.
I'll revert it.